src/api/amazonUser/amazonUser.js

```diff
@@ -0,0 +1,48 @@
+import logger from '@/common/utils/logger';
```

Comment:
This file has most of what was in the original Google Doc.  The weather function was separated into its respective file. I really wanted to convert it to typescript but ran out of time.